samples/nullnessUnspecifiedTypeParameter/nullnessunspecifiedtypeparameter/NullnessUnspecifiedTypeParameter.java

```diff
@@ -31,8 +31,8 @@ class Test {}
 class Instances {
   static final NullnessUnspecifiedTypeParameter<Object> A1 =
       new NullnessUnspecifiedTypeParameter<>();
-  // :: error: jspecify_nullness_mismatch
   static final NullnessUnspecifiedTypeParameter<@Nullable Object> A2 =
+      // :: error: jspecify_nullness_mismatch
```

Comment:
In the really old version of the CF `main` uses, the error was apparently on the `=`.
For a long while, it is now on the right-hand side of the assignment, which is here on a different line.
As these are tests for a particular tool, I think it's fine to move the expected error.
For the conformance tests, we should look into ensuring such tests are always on one line.

Reply:
Moving it SGTM.

Putting everything on one line in the future sounds good if we can get away with it.